src/status_im/ui/screens/wallet/choose_recipient/events.cljs

```diff
@@ -6,12 +6,12 @@
 (handlers/register-handler-db
   :wallet/toggle-flashlight
   (fn [db]
-    (let [flashlight-state (get-in db [:wallet/send-transaction :camera-flashlight])
+    (let [flashlight-state (get-in db [:wallet :send-transaction :camera-flashlight])
```

Comment:
`send-transaction` was in wallet ns, and it shouldn't be in wallet map, could you elaborate?

Reply:
two main reasons for me are
- it makes it easier to clear out when switching accounts
- less top-level subscriptions

Reply:
I agree with @yenda on this

Reply:
the idea to have ns keywords is to find usages , and :wallet/wallet for wallet screen,  :wallet/send-transaction for send-transaction screen, tbh can't see how it makes it harder to clear out